examples/Serving-An-Implicit-Model-With-Merlin-Systems.ipynb

```diff
@@ -0,0 +1,451 @@
+{
```

Comment:
<p>The logo is the xgboost-model-with-merlin-systems logo</p><p> </p><p><code>We train an XGBoost model to predict the rating based on user and item features and proceed to deploy it to the Triton Inference Server.</code>   it is XGBoost here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/systems/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
fixed